### ISSUE TITLE
bedmap count incorrect result

### ISSUE THREAD

**Issue by skytguuu** [bedops/bedops#232]
Hi,
Recently I tried to use bedmap to count the overlap of two bed.files.
One ref-bed file is like:
######
chr11	3000000	4000000
chr11	4000000	5000000
chr11	5000000	6000000
chr11	6000000	7000000
chr11	7000000	8000000
chr11	8000000	9000000
chr11	9000000	10000000
######
Another file is like:
######
chr11	4660000	4980000
chr11	5340000	5420000
chr11	6340000	6520000
chr11	9060000	9180000
######
However, when I tried to count the overlap numbers, the result showed zero.
command is :bedmap --echo --count ref.bed input.txt | sed 's/|/\t/g' > result.txt
the result is as followed:
#########
chr11	3000000	4000000	0
chr11	4000000	5000000	0
chr11	5000000	6000000	0
chr11	6000000	7000000	0
chr11	7000000	8000000	0
chr11	8000000	9000000	0
chr11	9000000	10000000	0
#########
 I am confused about this result. I also have tried use the bed format of 4 or 5 columns of input.txt, but the result is the same.  Could you give me a hand?

Thanks
Best

**Comment by sjneph** [comment]
Your inputs may have spaces instead of tabs.  Add --ec to do formal input error checking and --delim "\t" to change the default | delimiters to tabs.

**Comment by skytguuu** [comment]
As your advise, I add "--ec" to check the error. I found the error reported bed file not properly sorted. But I checked my ref.bed and input.txt, they are all sorted as " chr1,chr2...chr9,chr10,chr11...chr19". I also changed the spaces instead of tabs. The result is the same. But when I sorted those files as "chr1,chr10,chr11...". The result are correct. So, it seemed bedmap could only handle with the sorted bed file with "chr1,chr10,chr11...". That's why I have right count numbers in "chr1..chr9", but wrong numbers in "chr10...chr19".

Thanks for your help!
Best
